week-1/implement/sum.js

```diff
@@ -0,0 +1,15 @@
+function sum(list) {
+  if (list.length === 0) {
+    return 0;
+  } else {
+    let sumOfNum = 0;
```

Comment:
Is good practice to declare your variables at the top at the beginning of the function. SO can you remove the sumOfNum var to line 2?